chia/util/streamable.py

```diff
@@ -152,80 +155,121 @@ def streamable(cls: Any):
     """
 
     cls1 = strictdataclass(cls)
-    return type(cls.__name__, (cls1, Streamable), {})
+    t = type(cls.__name__, (cls1, Streamable), {})
+
+    parse_functions = []
+    try:
+        fields = cls1.__annotations__  # pylint: disable=no-member
+    except Exception:
+        fields = {}
+
+    for _, f_type in fields.items():
+        parse_functions.append(cls.function_to_parse_one_item(f_type))
+
+    PARSE_FUNCTIONS_FOR_STREAMABLE_CLASS[t] = parse_functions
+    return t
+
+
+def parse_bool(f):
+    bool_byte = f.read(1)
+    assert bool_byte is not None and len(bool_byte) == 1  # Checks for EOF
```

Comment:
Will these asserts activate in any run mode, or is it like C, where they can be deactivated in some run modes?

Reply:
`assert` is removed in `python -OO` mode, which we don't do. So we shouldn't use `assert` for flow control. However, this problem was here before this change, so it doesn't make these semantics any worse.